idris.cabal

```diff
@@ -264,7 +264,7 @@ Library
                 , cheapskate < 0.2
                 , containers >= 0.5 && < 0.6
                 , deepseq < 1.5
-                , directory >= 1.2.2.0 && < 1.2.3.0 || > 1.2.3.0
+                , directory > 1.2.3.0
```

Comment:
This bound should be `directory >= 1.2.3`, if I understand the situation correctly.